commands/Fun/rps.js

```diff
@@ -0,0 +1,34 @@
+const { Command } = require('klasa');
+const choices = ['rock', 'paper', 'scissors'];
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			description: 'Play a game of rock, paper, scissors with the bot.',
+			usage: '<move:string>'
+		});
+	}
+
+	async run(message, [move]) {
```

Comment:
Lets use action instead of move as action can be used in various commands giving a better consistency across the code